test/build/basic/basic.test.js

```diff
@@ -140,4 +140,14 @@ describe('bundle command', () => {
         expect(stderr).toContain("Run 'webpack --help' to see available commands and options");
         expect(stdout).toBeFalsy();
     });
+
+    it('should log supplied config when logging level is log', () => {
+        if (isWebpack5) {
```

Comment:
Should worn on v4 too

Reply:
worn?

Reply:
work :smile:

Reply:
This is introduced in core flag, was not supported before as flag

Reply:
It is just logger, please remove `isWebpack5`

Reply:
won't work 
![image](https://user-images.githubusercontent.com/21009455/107418025-2bae8700-6b3c-11eb-8ea5-c5534d06c890.png)